src/nvim/tui/tui.c

```diff
@@ -1857,10 +1857,7 @@ static int unibi_find_ext_bool(unibi_term *ut, const char *name)
 
 /// Determine if the terminal supports truecolor or not:
 ///
-/// 1. If $COLORTERM is "24bit" or "truecolor", return true
-/// 2. Else, check terminfo for Tc, RGB, setrgbf, or setrgbb capabilities. If
-///    found, return true
-/// 3. Else, return false
+/// If terminfo contains Tc, RGB, or both setrgbf and setrgbb capabilities, return true.
 static bool term_has_truecolor(TUIData *tui, const char *colorterm)
 {
   // Check $COLORTERM
```

Comment:
Why are there only comment changes?

Reply:
Not sure what happened there, but fixed now.